clevertap-pushtemplates/src/main/java/com/clevertap/android/pushtemplates/TemplateRenderer.kt

```diff
@@ -252,7 +251,7 @@ class TemplateRenderer : INotificationRenderer, AudibleNotification {
                         try {
                             ptJsonObj = JSONObject(ptJsonStr)
                         } catch (e: Exception) {
-                            Logger.v("Unable to convert JSON to String")
+                            PTLog.verbose("Unable to convert JSON to String")
```

Comment:
@Anush-Shand we need to use the Logger of the core SDK here? please confirm with @piyush-kukadiya